### ISSUE TITLE
Rubocop silently fails with small error in config file

### ISSUE THREAD

**Issue by smackesey** [rubocop-hq/rubocop#7725]
Rubocop seems to silently fail when there is a small typo in a configuration file.

--------

## Expected behavior

Rubocop should print an error message indicating that the config file did not parse correctly.

## Actual behavior

Rubocop runs but does not actually perform any linting.

## Steps to reproduce the problem

Create a directory with two files:

**.rubocop.yml**

```yml
LayoutEmptyLinesAroundArguments:
  Enabled: false
```

**test.rb**

```ruby
def badName
  if something
    test
    end
end
```

Now go to the directory and run `rubocop --debug test.rb`. You should see some debug output inidicating configuration loading but no linting output.

## RuboCop version

0.80.0 (using Parser 2.7.0.2, running on ruby 2.7.0 x86_64-darwin19)

**Comment by tejasbubane** [comment]
Output shows `unrecognized cop`. Should this be improved?

```
Inheriting configuration from /Users/tejas.bubane/opensource/rubocop/.rubocop_todo.yml
Error: unrecognized cop LayoutEmptyLinesAroundArguments found in .rubocop.yml
Finished in 0.2368809999898076 seconds
```

**Comment by smackesey** [comment]
@tejasbubane Hmm, I don't get this message. Here is my output:
```
For /Users/smackesey/Desktop/tester: configuration from /Users/smackesey/Desktop/tester/.rubocop.yml
Default configuration from /Users/smackesey/.asdf/installs/ruby/2.7.0/lib/ruby/gems/2.7.0/gems/rubocop-0.80.0/config/default.yml
Finished in 0.07307699997909367 seconds
```

**Comment by lxxxvi** [comment]
I also get the (expected) error

```
Error: unrecognized cop LayoutEmptyLinesAroundArguments found in .rubocop.yml
```

(Ruby 2.7.1, RuboCop 0.85.1)

**Comment by tejasbubane** [comment]
@smackesey Are you still getting that error with latest rubocop?

**Comment by smackesey** [comment]
Just tested again with rubocop 0.85.1, ruby 2.7.1. G

```
[I] ❯ rubocop --debug test.rb
For /Users/smackesey/Desktop/tester: configuration from /Users/smackesey/Desktop/tester/.rubocop.yml
Default configuration from /Users/smackesey/.asdf/installs/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/rubocop-0.85.1/config/default.yml
Finished in 0.07465199998114258 seconds
```

Still no error message for me.

**Comment by dvandersluis** [comment]
This will now (as of #9171, not released yet) output the following (it outputs on stderr so if you're still having this issue check that maybe you're not suppressing it):

```
Error: unrecognized cop LayoutEmptyLinesAroundArguments found in .rubocop.yml
Did you mean `Layout/EmptyLinesAroundArguments`?
```

I think this issue can be closed. (/cc @bbatsov)
